tests/posting/test_log2k8s.py

```diff
@@ -0,0 +1,88 @@
+import asyncio
+import io
+import logging
+
+import pytest
+
+import kopf
+from kopf.engines.logging import ObjectLogger
+from kopf.engines.logging import ObjectPrefixingFormatter
+
+OBJ1 = {'apiVersion': 'group1/version1', 'kind': 'Kind1',
+        'metadata': {'uid': 'uid1', 'name': 'name1', 'namespace': 'ns1'}}
+REF1 = {'apiVersion': 'group1/version1', 'kind': 'Kind1',
+        'uid': 'uid1', 'name': 'name1', 'namespace': 'ns1'}
+
+
+@pytest.fixture()
+def logstream(caplog):
+    """ Prefixing is done at the final output. We have to intercept it. """
```

Comment:
This fixture is almost identical to the one in `test_cause_logging.py`, except it sets `logger.handlers` back to its old value after the test (whereas the other fixture sets `logger.handlers` to the empty list afterwards). Why this difference?

Reply:
Overlooked that. Fixed. Now, the fixture is in top-level `conftest.py`.